library/src/main/java/com/android/mms/transaction/TransactionService.java

```diff
@@ -700,9 +705,12 @@ protected void endMmsConnectivity() {
             // cancel timer for renewal of lease
             mServiceHandler.removeMessages(EVENT_CONTINUE_MMS_CONNECTIVITY);
             if (mConnMgr != null && Build.VERSION.SDK_INT < Build.VERSION_CODES.M) {
-                mConnMgr.stopUsingNetworkFeature(
-                        ConnectivityManager.TYPE_MOBILE,
-                        "enableMMS");
+                try {
+                    java.lang.reflect.Method method = mConnMgr.getClass().getMethod("stopUsingNetworkFeature", int.class, String.class);
+                    method.invoke(ConnectivityManager.TYPE_MOBILE, "enableMMS");
+                } catch (Exception e) {
+                    Log.v(TAG, "failed to invoke mConnMgr.stopUsingNetworkFeature: enableMMS", e);
+                }
```

Comment:
These methods were wholesale removed before Android 14, but it's my understanding that neither Light Phone device will actually call these, because the library defaults to using system sending, which means this codepath never runs unless you opt into it.

So, to get the lib to compile on both flavors I'm using reflection to access these methods.